ejb-timer/.github/quickstart_ejb_timer_ci.yml

```diff
@@ -0,0 +1,21 @@
+name: WildFly EJB Timer Quickstart CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+      - <quickstart-path>/**'
+      - .github/workflows/quickstart_ci.yml
+
+jobs:
+  call-quickstart_ci:
+    uses: ./.github/workflows/quickstart_ci.yml
+    with:
+      QUICKSTART_PATH: ejb-timer
+      DEPLOYMENT_DIR: webapp
+      TEST_PROVISIONED_SERVER: true
+      TEST_BOOTABLE_JAR: true
+      TEST_OPENSHIFT: true
+      SERVER_PROVISIONING_SERVER_HOST: https://localhost:8443
+      MATRIX_JDK: '"17"'
```

Comment:
here you are reducing the matrix to jdk 17 and linux only, which is not intentional I think. I see no reason for this QS CI to not run on default matrix (JDK: 11, 17; OS: linux, windows) so please remove the MATRIX attributes